backend/models/User.js

```diff
@@ -20,14 +20,13 @@ const userSchema = new mongoose.Schema({
   lastName: { type: String, required: true, minlength: 2, maxlength: 20 },
   dob: { type: Date, required: true },
   gender: { type: String, required: true },
-  nationality: [String],
-  photo: String,
-  newcomer: Boolean,
-  volunteer: Boolean,
+  nationality: String,
```

Comment:
this was an array on purpose..to deal with people who might have multiple nationalities..
nationality: [String],

Reply:
As mentioned before, I've never seen multiple nationalities in a registration form on websites, this is why I changed it

Reply:
ok :)